src/wield/utilities/file_io/any_io.py

```diff
@@ -213,22 +213,22 @@ def write_any(
     fdict,
 ):
     features = types.type2features[ftype]
-    fdict_orig = None
+    # fdict_orig = None
 
-    if fdict_orig is None:
-        fdict_orig = fdict
-        fdict = copy.deepcopy(fdict)
+    # if fdict_orig is None:
```

Comment:
`fdict_orig` was always `None` here so it was always `fdict` after this if statement